pkg/reconciler/pubsub/pullsubscription/static/controller.go

```diff
@@ -80,8 +83,10 @@ func NewController(
 	r := &Reconciler{
 		Base: &psreconciler.Base{
 			PubSubBase:             pubsubBase,
+			Identity:               identity.NewIdentity(ctx),
 			DeploymentLister:       deploymentInformer.Lister(),
 			PullSubscriptionLister: pullSubscriptionInformer.Lister(),
+			ServiceAccountLister:   serviceAccountInformer.Lister(),
```

Comment:
I think you want the serviceAccountLister in the Identity object instead of thihs psreconciler?

Reply:
Yes...I'll remove that, and create a new issue for it.